### ISSUE TITLE
upgrade projects to 0.7.5.1?

### ISSUE THREAD

**Issue by dmcweeney** [nikhilk/scriptsharp#220]
Hi,

I've a VS 2010 solution with 10 Script# projects in it - currently using Script# 0.7.4.0.

I'm need to upgrade this to VS 2012 and accordingly Script# 0.7.5.1. I see that the new Script# is based on nuget packages etc and there are local copies of the package files stored at the solution level with Import directives to these contents. I've never done anything with nuget packages so I'm totally unfamiliar with it.

I'm assuming there wont be an script# msi for 0.7.5.1 so I was wondering what is the best approach for upgrading the existing projects I have. Manually compare the csproj filese and fixup accordingly?

Thanks

Donal

**Comment by michaelaird** [comment]
I've started to document the update process here:

https://github.com/nikhilk/scriptsharp/issues/216#issuecomment-7373982

read through the rest of the comments as there's some valuable feedback from Nikhil.

**Comment by dmcweeney** [comment]
Hi Michael,

Thanks for the link - it was a big help in getting this sorted.

I did notice that VS 2012 RC was a bit flakey and I had to restart it a few times to get the projects to reload after making the changes to the .csproj files.

Another issue I noticed was that after adding the Script# nuget package to a project using the "Manage Nuget packages for Solution" a file called ScriptSharp.PlaceHolder.txt would be added to project source control but this file was not added to the project files. ie TFS thought there was a file of this name added. I did an undo on this action and all seems well. Everything builds now anyway.

Thanks

Donal
